src/pps/pps.proto

```diff
@@ -268,54 +270,50 @@ message StoredJobInfo {
   // Download/process/upload time and download/upload bytes
   ProcessStats stats = 10;
 
-  pfs_v2.Commit stats_commit = 11;
-  JobState state = 12;
-  string reason = 13;
-  google.protobuf.Timestamp started = 14;
-  google.protobuf.Timestamp finished = 15;
+  JobState state = 11;
+  string reason = 12;
+  google.protobuf.Timestamp started = 13;
+  google.protobuf.Timestamp finished = 14;
 }
 
 message JobInfo {
```

Comment:
Would it make sense to just nest the `StoredJobInfo` rather than copy the fields over?

Reply:
Ideally we could inline all the fields, but I don't think that's possible, and I don't think we want to sacrifice ergonomics for this.